samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java

```diff
@@ -148,6 +148,41 @@ public void assignMFAGroupBeforeScenario(Scenario scenario) {
 		groupList.forEach(group -> Page.getUser().addToGroup(group.getId()));
 	}
 
+	@Before("@requireWebAuthnRequiredGroupsForUser")
+	public void assignWebAuthnRequiredBeforeScenario(Scenario scenario) {
+		Assert.assertNotNull(Page.getUser());
+		List<String> groups = new ArrayList<>();
+		groups.add("WebAuthn Required");
+
+		List<Group> groupList = client.listGroups()
+				.stream()
+				.filter(group -> groups.contains(group.getProfile().getName()))
+				.collect(Collectors.toList());
+		Assert.assertFalse(groupList.isEmpty());
+		groupList.forEach(group -> Page.getUser().addToGroup(group.getId()));
+	}
+	public void assignWebAuthnGroup() {
```

Comment:
nit: add empty line after method ends and new method starts

Reply:
Added empty line